src/components/postFooter/GifContainer.jsx

```diff
@@ -28,8 +28,8 @@ const GifContainer = ({ displayTenor, removeTenor, handleSelectedGif }) => {
   const searchTenor = async (e) => {
     const lmt = 10;
     const searchValue = e.target.value;
-    const category = document.querySelector(".categoriesSection");
-    const singleCategory = document.querySelector(".singleCategory");
+    const category = window.document.querySelector(".categoriesSection");
+    const singleCategory = window.document.querySelector(".singleCategory");
     // Showing the results container if and only if you're currently searching.
```

Comment:
if we need to add comments why don't we make the variables more expressible of the intent?

Reply:
I think the variable name is expressive of intent I just wanted to try out commenting my work. I can get rid of them if its a a problem

Reply:
awesome, I guess we can do without the comment then.
@Izima01